src/include/binder/copy/bound_copy.h

```diff
@@ -13,15 +14,24 @@ namespace binder {
 
 class BoundCopy : public BoundStatement {
 public:
+    // COPY FROM
```

Comment:
Since you have CopyTo and CopyFrom in the parser part. Let's use BoundCopyFrom and BoundCopyTo for consistency.